src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/alerts/AlertsResource.kt

```diff
@@ -0,0 +1,136 @@
+  @Operation(
+    summary = "get an alert by bookingId and alert sequence",
+    description = "Retrieves an prisoner alert. Requires ROLE_NOMIS_ALERTS",
+    responses = [
+      ApiResponse(
+        responseCode = "200",
+        description = "Alert Information Returned",
+        content = [
+          Content(
+            mediaType = "application/json",
+            schema = Schema(implementation = AlertResponse::class),
+          ),
+        ],
+      ),
+      ApiResponse(
+        responseCode = "401",
+        description = "Unauthorized to access this endpoint",
+        content = [
+          Content(
+            mediaType = "application/json",
+            schema = Schema(implementation = ErrorResponse::class),
+          ),
+        ],
+      ),
+      ApiResponse(
+        responseCode = "403",
+        description = "Forbidden to access this endpoint. Requires ROLE_NOMIS_ALERTS",
+        content = [
+          Content(
+            mediaType = "application/json",
+            schema = Schema(implementation = ErrorResponse::class),
+          ),
+        ],
+      ),
+      ApiResponse(
+        responseCode = "404",
+        description = "Alert does not exist",
+        content = [
+          Content(
+            mediaType = "application/json",
+            schema = Schema(implementation = ErrorResponse::class),
+          ),
+        ],
+      ),
+    ],
+  )
+  fun getAlert(
+    @Schema(description = "Booking Id", example = "12345")
+    @PathVariable
+    bookingId: Long,
+    @Schema(description = "Alert sequence", example = "3")
+    @PathVariable
+    alertSequence: Long,
+  ): AlertResponse = alertsService.getAlert(bookingId, alertSequence)
+}
+
+@Schema(description = "The data held in NOMIS about an alert associated with a prisoner")
+data class AlertResponse(
+  @Schema(description = "The prisoner's bookingId related to this alert")
+  val bookingId: Long,
+  @Schema(description = "The sequence primary key within this booking")
+  val alertSequence: Long,
+  @Schema(description = "The alert code")
+  val alertCode: CodeDescription,
+  @Schema(description = "The alert type")
+  val type: CodeDescription,
+  @Schema(description = "Date alert started")
+  val date: LocalDate,
+  @Schema(description = "Date alert expired")
+  val expiryDate: LocalDate? = null,
+  @Schema(description = "true if alert is active and has not expired")
+  val isActive: Boolean = true,
+  @Schema(description = "true if alert has been verified by another member of staff")
+  val isVerified: Boolean = false,
+  @Schema(description = "Free format text of person or department that authorised the alert", example = "security")
+  val authorisedBy: String? = null,
+  @Schema(description = "Free format comment")
+  val comment: String? = null,
+  @Schema(description = "Audit data associated with the records")
+  val audit: NomisAudit,
+)
+
+@Schema(description = "The data held in NOMIS the person or system that created this record")
```

Comment:
keeping audit data separate since I'm not convinced DPS need this even though they have asked for it. Same applies to DSL